salt/search/config/etc-elasticsearch-elasticsearch.yml

```diff
@@ -0,0 +1,6 @@
+# Adapted from: https://github.com/elifesciences/builder-base-formula/blob/master/elife/config/etc-elasticsearch-elasticsearch.yml
+# Elasticsearch configuration for {{ grains['fqdn'] }}
+
+node.name: "{{ grains['fqdn'] }}"
```

Comment:
yup. it looks like it's after a unique id in case of clustering, so something like 'search--end2end--1' would be fine